aa-sdk/core/src/signer/local-account.ts

```diff
@@ -95,6 +96,33 @@ export class LocalAccountSigner<
     return this.inner.signTypedData(params);
   };
 
+  /**
+   * Signs an unsigned authorization using the provided private key account.
+   *
+   * @example
+   * ```ts
```

Comment:
nit: can we add `twoslash` after the `ts` here?

At some point we'll want to do this for ALL examples retroactively as well, but that will cause too many build breaks I fear

Reply:
added and validated!